README.md

```diff
@@ -87,8 +102,8 @@ if you want to work more rapidly.
 
 If you configure an environment variable called `EDITOR`, either on your
 system as a whole or in the root `.env` file, it can be used in the development
-server to link to sources which, when clicked, opens in
-your preferred editor/IDE. For example, in the root:
+server to link to sources which, when clicked, open in your preferred
+editor/IDE. For example, in the root of the repo you could run:
 
     echo 'EDITOR=code' >> .env
```

Comment:
I really wish we had an example that Windows people could use. 
`code` is a weird word for Microsoft Visual Studio Code. People might think it's a generic term. 

Most people on Windows use some sort of "bash" prompt. Perhaps even the one built into VSCode. And if they suse that `echo` and `>>` will work. 

@hamishwillee Do you have an equivalent for Windows we can show as an example here?

Reply:
Notepad++ is quite popular on Windows.

Reply:
CMD / PowerShell (with a tendency to the latter) is shown in VS Code on Windows as far as I remember.

Reply:
I think the text is OK as is WRT code being maybe seen as a generic term.

but I would like to discuss your comment `And if they suse that echo and >> will work.`, @peterbe . Are you worried that Windows users won't be able to get the commands to work? Perhaps it is  wise to put some kind of note near the top of the article saying "Note: Windows users might have trouble running some of the commands listed in this README. You need to make sure you are running an environment that supports these commands, such as Git Bash or Powershell".

What do yout think @hamishwillee ?